tests/provision_test.go

```diff
@@ -70,13 +74,19 @@ func fsVolCreationTest() {
 		verifyPVForPVC(false, pvcName)
 		By("Deleting storage class", deleteStorageClass)
 	}
+	removeVg("lvmvg")
```

Comment:
this should be done with a defer

Reply:
or perhaps better to use beforeEach and afterEach kind of thing?

Reply:
I have suggested not doing PV and VG setup for each test. Rather setup once at ginkgo entry point, use for all tests, and cleanup at the end. Only if a test needs anything additional can perform those additional setup from that test.

Reply:
Yes doing cleanup and setup in the suite level